bitsandbytes/__main__.py

```diff
@@ -63,7 +63,7 @@ def generate_bug_report_information():
         print('')
 
     print_header("LD_LIBRARY CUDA PATHS")
-    lib_path = os.environ['LD_LIBRARY_PATH'].strip()
+    lib_path = os.environ['LD_LIBRARY_PATH'].strip() if 'LD_LIBRARY_PATH' in os.environ else ''
```

Comment:
```suggestion
    lib_path = os.environ.get('LD_LIBRARY_PATH', '').strip()
```

How about using `dict.get()` rather than `key in dict` if it is to get a value?